docs/Polystat-docs-options.md

```diff
@@ -1,123 +1,99 @@
+- **Automate Font Color**: Uncheck "Automate Font Color" to manually set the font color. The automated option uses the current theme to pick a color, which may not be suitable for all cases.
 
 ![Polygon Text Manual Font Color](https://raw.githubusercontent.com/grafana/grafana-polystat-panel/v2.x/src/img/screenshots/polystat-v2-text-manual-font-color.png)
 
-Manually Set Font Color with color picker
+You can manually set Font Color with the color picker.
 
 ![Polygon Text Font Manual Color Picker](https://raw.githubusercontent.com/grafana/grafana-polystat-panel/v2.x/src/img/screenshots/polystat-v2-text-font-color-picker.png)
 
 ## Sorting
 
-The order (left to right) of the displayed polygons can be set with the sort options.
+Sorting allows you to set the order (left to right) of the displayed polygons.
 
 ![Sort Settings](https://raw.githubusercontent.com/grafana/grafana-polystat-panel/v2.x/src/img/screenshots/polystat-v2-sorting.png)
 
-The following directions are supported:
+Supported directions:
 
 ![Sorting Directions](https://raw.githubusercontent.com/grafana/grafana-polystat-panel/v2.x/src/img/screenshots/polystat-v2-sorting-directions.png)
 
-And the following fields:
+Supported fields:
 
 ![Sorting Fields](https://raw.githubusercontent.com/grafana/grafana-polystat-panel/v2.x/src/img/screenshots/polystat-v2-sorting-fields.png)
 
 ## Tooltips
 
 ![Tooltips](https://raw.githubusercontent.com/grafana/grafana-polystat-panel/v2.x/src/img/screenshots/polystat-v2-tooltips-all.png)
 
-### Enable/Disable Tooltip
-
-Toggles displaying tooltips for the panel.
-
-### Tooltip Font Family
+The following options are available:
 
-Sets the font family to be used in tooltips.
+- **Enable/Disable Tooltip**: Toggles displaying tooltips for the panel.
 
-### Show Timestamp
+- **Tooltip Font Family**: Sets the font family to be used in tooltips.
 
-Toggles display of the timestamp at the bottom of the tooltip.
+- **Show Timestamp**: Toggles display of the timestamp at the bottom of the tooltip.
 
-### Display mode
-
-You can choose to display only metrics that have triggered a threshold in the tooltip, or display all metrics. This is useful when there are many metrics rolled up into a composite.
+- **Display mode**: Displays either only metrics that have triggered a threshold in the tooltip or all metrics. This is useful when there are many metrics rolled up into a composite.
 
 ![Tooltip Display Modes](https://raw.githubusercontent.com/grafana/grafana-polystat-panel/v2.x/src/img/screenshots/polystat-v2-tooltips-display-modes.png)
 
-### Tooltip - Non Triggered State Text
-
-When there are no threshold violations, this text will be displayed in the tooltip instead of the metric value.  Leave blank if you want to show the value.
+- **Non Triggered State Text**: When there are no threshold violations, displayes this text in the tooltip instead of the metric value. Leave blank if you want to show the value.
```

Comment:
```suggestion
- **Non Triggered State Text**: When there are no threshold violations, display this text in the tooltip instead of the metric value. Leave blank if you want to show the value.
```